backend/internal/handlers/classrooms/assignments/works/works.go

```diff
@@ -55,6 +56,16 @@ func (s *WorkService) getWorksInAssignment() fiber.Handler {
 			return errs.BadRequest(err)
 		}
 
+		assignmentOutline, err := s.store.GetAssignmentByID(c.Context(), int64(assignmentID))
+		if err != nil {
+			return errs.InternalServerError()
```

Comment:
Same thing as above, can this be a table join

Reply:
I need both later on, so I'm keeping this one